lib/code_corps/model/skill.ex

```diff
@@ -20,7 +20,13 @@ defmodule CodeCorps.Skill do
   @doc """
   Builds a changeset based on the `struct` and `params`.
   """
-  def changeset(struct, params \\ %{}) do
+  @spec create_changeset(CodeCorps.Skill.t, map) :: Ecto.Changeset.t
+  def create_changeset(struct, params \\ %{}) do
+    changeset(struct, params)
+  end
+
+  @spec changeset(CodeCorps.Skill.t, map) :: Ecto.Changeset.t
+  defp changeset(struct, params \\ %{}) do
```

Comment:
I think this just adds an extra layer we don't really need. I see why you did it, but I think it would be preferable to revert on this and instead just call `Skill.changeset` from the controller.